docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
-   - Due to the size constraints, the connector will be phased out by the Cost Management team starting in 2024.
-   - The APIs do not include some key columns for savings plans, like the BenefitId/Name columns. All costs are covered but not always easily identifiable.
-2. For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use raw exports with Power BI.
-3. For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use FinOps hubs with Power BI.
+1. **Power BI on Cost Exports**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use raw exports with Power BI.
+2. **FinOps Hubs with Azure Data Factory**: For costs under $2-5M/month (~$65M total)<sup>2</sup> that need savings plan data, you can use FinOps hubs with Power BI.
+3. 🆕 **FinOps Hubs with Azure Data Explorer (ADX)**:  For scenarios where you need more advanced exploration, Hubs on ADX allows you to leverage Kusto Query Language (KQL) for in-depth analysis and provides built-in queries to simplify data exploration. ADX also offers scalability, allowing you to increase capacity by adjusting the virtual machine size. The default VM size supports up to $10M/month of cost data.
 4. _**Coming soon:**_ For costs over $5M/month or for additional capabilities, you can connect Fabric to either FinOps hubs or raw exports.
    - This is possible today, but is not supported in FinOps toolkit reports yet. Support will be added in a future release.
 
 _<sup>1) Power BI Pro can handle ~$2M of raw cost data. Power BI Premium can handle ~$5M.</sup>_
 
 _<sup>2) The $2-5M limits are for Power BI data refreshes and apply on a monthly basis for hubs and raw exports. They can load up to $65M with incremental refresh enabled.</sup>_
 
-In general, we recommend starting with the Cost Management connector when getting started with Power BI reports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
+_<sup>3) ❗ The Cost Management connector is no longer recommended as it will be deprecated soon.</sup>_
 
-| Capabilities                                        |            Connector             |             Exports              |           FinOps hubs            | Microsoft Fabric<sup>1</sup> |
+In general, we recommend starting with Power BI reports by connecting to the cost exports. The most common reasons to switch to FinOps hubs are for additional account types and scopes or to enable more advanced capabilities. Use the following comparison to help you make the decision:
+
+| Capabilities                                        |             Power BI on Cost Exports              |        FinOps hubs with ADF         |        FinOps hubs with ADF and ADX         | Microsoft Fabric<sup>1</sup> |
```

Comment:
The further down this list I get, the more I wonder if we should even call out hubs on storage 🤔